examples/archive.html

```diff
@@ -1,65 +1,135 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
   <head>
     <title>MapKnitter Lite</title>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"/>
-    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
-    <link rel="stylesheet" href="../node_modules/leaflet/dist/leaflet.css">
-    <link rel="stylesheet" href="../dist/vendor.css">
-    <link rel="stylesheet" href="../dist/leaflet.distortableimage.css" media="screen" title="no title">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-
-    <script src="../node_modules/leaflet/dist/leaflet.js" type="text/javascript" charset="utf-8"></script>
-    <script src="../dist/vendor.js"></script>
-    <script src="../dist/leaflet.distortableimage.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
-    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <meta charset='utf-8' />
+    <meta
+      name='viewport'
+      content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no'
+    />
+    <meta http-equiv='x-ua-compatible' content='ie=edge' />
+    <link rel='stylesheet' href='../node_modules/leaflet/dist/leaflet.css' />
+    <link rel='stylesheet' href='../dist/vendor.css' />
+    <link
+      rel='stylesheet'
+      href='../dist/leaflet.distortableimage.css'
```

Comment:
Hmm it still looks like there are new lines and spaces added here. Could you leave any unrelated lines as they were? Thank you so much!

Reply:
Hello @jywarren I'll get on it, in the mean time could you check [this out](https://github.com/publiclab/Leaflet.DistortableImage/issues/665), i and @THEDARKMAN007 need some information. 
Thank you!